src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -277,10 +270,35 @@ def retrieve_assembly_data(
 def main() -> None:
     """Module's entry-point."""
     parser = ArgumentParser(description="Download an assembly data files from INSDC or RefSeq.")
+    parser.add_argument("-v", "--verbose", action="store_true", required=False,
+        help="Verbose level logging")
+    parser.add_argument("-d", "--debug", action="store_true", required=False,
+        help="Debug level logging")
     parser.add_argument("--accession", required=True, help="Genome assembly accession")
     parser.add_argument_dst_path(
         "--download_dir", default=Path.cwd(), help="Folder where the data will be downloaded"
     )
     args = parser.parse_args()
 
-    retrieve_assembly_data(**vars(args))
+    # Configure logging
+    log_file = ("assembly_download.log")
```

Comment:
I guess we don't need to have the log specifically to a file in this script. Since there is not much log we can get when "only" downloading some files, stderr should be enough.

Reply:
Done, logging goes to STDERR